src/redisenterprise/azext_redisenterprise/manual/_params.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    get_enum_type
+)
+from ..action import (
+    AddPersistence,
+    AddModules
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('redisenterprise create') as c:
+        # Update help
+        c.argument('sku', arg_type=get_enum_type(['Enterprise_E10', 'Enterprise_E20', 'Enterprise_E50',
+                                                  'Enterprise_E100', 'EnterpriseFlash_F300', 'EnterpriseFlash_F700',
+                                                  'EnterpriseFlash_F1500']), help='The type of RedisEnterprise cluster '
```

Comment:
try to get enum values from SDK as defined in with `get_enum_type(SkuName)`:
https://github.com/Azure/azure-cli-extensions/blob/db558a6ffc172f7f6e3ac13b062857f804457f32/src/redisenterprise/azext_redisenterprise/vendored_sdks/redisenterprise/models/_redis_enterprise_management_client_enums.py#L139-L150

Reply:
Fixed